docker/Dockerfile.rocm-complete

```diff
@@ -0,0 +1,68 @@
+# Docker container for running DGL on AMD GPUs with ROCM, including DGL itself.
+
+# TODO: Rebuilding DGL is slow. Consider multistage builds and shared cache
+# mounts.
+
+FROM gcmn/dgl:rocm-deps-only
+
+# By default, build for all the architectures that the rocm-deps-only base image
+# is built for (limited by hip collections). Building for multiple
+# archictectures dramatically slows down a subsequent DGL build, especially of
+# graphbolt, but if we want a single image that "just works" for users, then I
+# think this is what we have to do :-/
+
+ARG GPU_TARGETS="gfx908;gfx90a;gfx940;gfx941;gfx942"
```

Comment:
Can you add a short line on the README about specifying a single architecture if one wants for faster builds?